src/khoj/processor/conversation/prompts.py

```diff
@@ -1029,6 +1043,13 @@
 """.strip()
 )
 
+additional_program_context = PromptTemplate.from_template(
+    """
+Here's some additional context about what happened while I was executing this query:
+{context}
```

Comment:
Ah no, this is more about aggregating any relevant information about errors encountered during execution. Think, as if it got system notifications about its own processes.

Reply:
I see, should we still simplify the prompt somewhat and remove the first person reference? It may make the who did what less confusing (as that can be inferred from message.role rather than the prompt)

```suggestion
Here are some additional results from the query execution:
{context}
```